store/index.php

```diff
@@ -49,7 +49,7 @@
             </div>
             <div class="icon-text">
                 <h3>You are missing API keys</h3>
-                <p>You are viewing a developmental version of the store without configuring api keys. This will lead to false positives and incorrect errors. Please set your keys to testing configuration.</p>
+                <p>You are viewing a development version of the store without configuring API keys. This will lead to false positives and incorrect errors. Please set your keys to testing configuration.</p>
```

Comment:
While I'm here, cleaning up some copy. This is only shown for local development anyway, but it was bugging me. :sweat: